.eslintrc

```diff
@@ -90,6 +90,13 @@
       "error",
       "always"
     ],
+    "flowtype/space-after-type-colon": [
+      2,
+      "always",
+      {
+        "allowLineBreak": true
```

Comment:
I needed to add `allowLineBreak` to be available to do 
https://github.com/kiwicom/smart-faq/pull/560/files#diff-7675e72b59cb13d5f92a71de8870e806R24